src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -247,7 +249,8 @@ TypeAnnotations BuildTypeAnnotations (TypeDefinition type)
 						}
 
 						if (annotatedMethods.Any (a => a.Method == setMethod)) {
-							_context.LogWarning ($"Trying to propagate DynamicallyAccessedMemberAttribute from property '{property.FullName}' to its setter '{setMethod.GetDisplayName ()}', but it already has such attribute on the 'value' parameter.", 2043, setMethod);
+							_context.LogWarning ($"Trying to propagate DynamicallyAccessedMemberAttribute from property '{property.FullName}' to its setter '{setMethod.GetDisplayName ()}', but it already has such attribute on the 'value' parameter.",
+								2043, setMethod, subcategory: MessageSubCategory.TrimCorrectness);
```

Comment:
Same as above - this is about correct usage of the attribute, not about trim correctness.